m1-runner/tasks/main.yml

```diff
@@ -0,0 +1,48 @@
+---
+# chia-runner/tasks/main.yml
+
+- name: Insert the proper path for gnu-tar in .zshenv.
+  lineinfile:
+    path: "~/.zshenv"
+    line: export PATH="/opt/homebrew/opt/gnu-tar/libexec/gnubin:$PATH"
+    insertbefore: EOF
+
+- name: Source the ~/.zshenv file
+  - shell: source ~/.zshenv
+
+- name: Make action-runner directory
+  ansible.builtin.file:
+    path: "{{ path_to_service }}"
+    state: directory
+    mode: '0755'
+
+- name: Download runner tarball
+  get_url:
+    url: https://github.com/actions/runner/releases/download/v{{runner_release_version}}/actions-runner-osx-x64-{{runner_release_version}}.tar.gz
+    dest: "{{ path_to_service }}"
+    mode: '0755'
+
+- name: Unarchive runner tarball
+  when: not service_script.stat.exists
+  ansible.builtin.unarchive:
+    src: "{{ path_to_service }}/actions-runner-osx-x64-{{runner_release_version}}.tar.gz"
+    dest: "{{ path_to_service }}"
+    remote_src: yes
+    list_files: yes
+    creates: "{{ path_to_service }}/actions-runner/svc.sh"
+
+- name: Configure the runner for GitHub
+  when: not service_script.stat.exists
+  shell: ./config.sh --url https://github.com/Chia-Network --token {{registration_token}} --runnergroup "M1 Runners" --labels m1 --name m1-01 --unattended
+  args:
+    chdir: "{{ path_to_service }}"
+    creates: "{{ path_to_service }}/actions-runner/svc.sh"
+
+- name: Install the runner service
```

Comment:
is there a creates or else a condition we can use to only run the "install" when we need to? What happens if we run install and its already installed?